README.md

```diff
@@ -1,38 +1,55 @@
-# PLSC
+# PLSC (Paddle Large Scale Classification)
 
 ## 1. Introduction
 
-[PLSC](https://github.com/PaddlePaddle/PLSC) is an open source Paddle Large Scale Classification Tools, which supports 60 million classes on single node 8 NVIDIA V100 (32G).
+[PLSC](https://github.com/PaddlePaddle/PLSC) is an open source Paddle Large Scale Classification Tools, which supports 92 million classes on single node 8 NVIDIA V100 (32G) and has high training throughtput. It implements [ArcFace](https://arxiv.org/abs/1801.07698), [CosFace](https://arxiv.org/abs/1801.09414), [PartialFC](https://arxiv.org/abs/2010.05222), SparseMomentum, DataParallel + ModelParallel distributed training, FP16 training.
```

Comment:
1. is an open source repo for a collection of Paddle Large Scale Classification Tools,
2. supports 92 million classes on a single node with 8 NVIDIA V100 (32G) GPUs